### DISCUSSION TITLE
Key Combos

### DISCUSSION THREAD

**Discussion by philipt18** [nickcoutsos/keymap-editor#54] [General]
How do I enter a key-combo like Command-Option-Shift-V (which is Paste and Match Style in an app I use)?

I see ZMK can do something like &kp LCMD(LALT(LSHFT(V))) but I don't know how to enter this in the editor.

And is that the same thing as setting the first three as modifiers and tapping the V? I assume that would use &mp although I'm not clear how to do that either.

Thanks.

**Comment by philipt18** [comment]
Maybe that would properly be: &kp LG(LA(LS(V)))

Based on: https://zmk.dev/docs/codes/modifiers#modifier-functions

How do I get that into the editor?

**Reply by Dimentium** [reply]
![ttt](https://user-images.githubusercontent.com/5391420/193783591-b9230d43-496c-4f8a-bbd0-92e40acfdd8d.gif)

**Reply by philipt18** [reply]
Awesome, thank you.

**Reply by nickcoutsos** [reply]
That's correct, `LS`/`LC`/`LA`/`LG` are the versions of modifiers that can be applied directly to a regular key code but understandably this can be a bit confusing if you're not looking for it specifically. This is something I'm thinking about revisiting in the near future as an enhancement to the picker for keycodes.

Thanks @Dimentium for sharing that :)

**Reply by jackytrabbit** [reply]
I really got surprised that we can do that in keymap editor...Nice feature

**Reply by osdiab** [reply]
is it possible to do this with `&mt` bindings? doesn't seem like it.

**Reply by nickcoutsos** [reply]
Not at the moment. When I initially added `&mt` it was with the understanding that the first parameter was a single modifier, but the implementation of the behavior technically allows for any keycode (which includes modified modifiers, like `LC(LALT)` for example).

I should probably just change it to allow any keycode, but in the short term you can get around this by making your own hold-tap behavior that uses `bindings =  <&kp>, <&kp>;` and it will interpret that to be keycodes.

**Reply by osdiab** [reply]
I ended up just tapping it in myself into the key map file but I guess from here on out the editor will wipe out my binding lol

**Reply by nickcoutsos** [reply]
It Shouldn't™.

Unless you touch that specific key's binding (not sure if changing the tap parameter would trigger this) the keymap editor generally will just leave it alone and complain with red highlighting that it doesn't seem to be a correct value type.
